spec/tic_tac_toe/board_spec.clj

```diff
@@ -3,24 +3,23 @@
             [tic-tac-toe.board :refer :all]))
 
 (def empty-board-3x3
-  [" " " " " "
-   " " " " " "
-   " " " " " "])
+  [space space space
+   space space space
+   space space space])
 (def empty-board-4x4
-  [" " " " " " " "
-   " " " " " " " "
-   " " " " " " " "
-   " " " " " " " "])
-(def X "X")
-(def O "O")
+  [space space space space
+   space space space space
+   space space space space
+   space space space space])
+
```

Comment:
This added space makes it more readable, but I would say consistency is most important.